### PR TITLE
Kong(official) package example

### PR DESCRIPTION
**Pull request by shashiranjan84**
As part  of https://github.com/mesosphere/universe/pull/1198

### CODE REVIEW

File: kong/1.9/README.md [hunk 1]
```diff
@@ -1,112 +1,203 @@
...
+    ![Kong on DC/OS](img/kong-dcos.png)
+  
+6. **Using Kong**
+    
+    Now that Kong is installed, curl the marathon-lb endpoint to validate that
+    it is up and running:
 
-![Kong on DC/OS](img/kong-dcos.png)
+    ```bash
+    $ curl <public-agent-ip>:10201
```
Comment (ryadav88) [inline kong/1.9/README.md]: I get `{"message":"no API found with those values"}`

File: kong/1.9/README.md [hunk 2]
```diff
@@ -1,112 +1,203 @@
...
+    ```
```
Comment (ryadav88) [inline kong/1.9/README.md]: Please add any instructions on how to actually use it.

File: kong/1.9/README.md [hunk 3]
```diff
@@ -1,112 +1,203 @@
...
+2. **Deploy Marathon-LB**
+
+    We will use [Marathon-LB](https://dcos.io/docs/1.9/networking/marathon-lb/)
+    for load balancing external traffic to cluster and
+    [VIPs](https://dcos.io/docs/1.9/networking/load-balancing-vips/virtual-ip-addresses/)
+    for load balancing internal traffic. Using the package `marathon-lb` deploy
+    the Marathon-LB:
+
+    ```bash
+    $ dcos package install marathon-lb
```
Comment (ryadav88) [inline kong/1.9/README.md]: why are you installing marathon-lb if youre not setting vhost afterwards?

File: kong/1.9/README.md (hunk 1 above)
Comment (shashiranjan84) [inline kong/1.9/README.md]: Its a valid response, a quick start-up link is provided in that section.

File: kong/1.9/README.md (hunk 2 above)
Comment (shashiranjan84) [inline kong/1.9/README.md]: same ^

File: kong/1.9/README.md (hunk 3 above)
Comment (shashiranjan84) [inline kong/1.9/README.md]: For external traffic, Kong package provides option to set vhost.

Comment (ryadav88) [inline kong/1.9/README.md]: please update the example to reflect that.

File: kong/1.9/README.md (hunk 1 above)
Comment (ryadav88) [inline kong/1.9/README.md]: update the example to say that this is a valid response.
And where does the user run Kong command? Master, node where its installed?

**Comment by shashiranjan84** [comment]
@ryadav88 Updated PR as advised.

**Pull request merged**